playbooks/azure/openshift-cluster/build_base_image.yml

```diff
@@ -0,0 +1,76 @@
+---
+# Note: this required ansible-galaxy install Azure.azure_modules
+
+- import_playbook: prerequisites.yml
+
+# create all the items necessary to make the VM
+- name: "Create resource group"
+  hosts: localhost
+  gather_facts: no
+  tasks:
+  - name: "provision public ip address"
+    import_role:
+      name: openshift_azure
+      tasks_from: provision_instance.yml
+    vars:
+      openshift_azure_vm_base_image:
+        name: '{{ openshift_azure_root_image }}'
+        resource_group: '{{ openshift_azure_root_image_ns }}'
+
+- name: update OS and install packages
+  hosts: azure
+  gather_facts: yes
+  remote_user: "{{ openshift_azure_vm_admin_username }}"
+  become: True
+  tasks:
+  - name: lay down yum certs
```

Comment:
is it worth adding a comment here that the certs are removed below (and another below that it is removing certs laid down above)?

Reply:
Added and refactored to its own file.